python/pyrogue/_PollQueue.py

```diff
@@ -41,11 +41,11 @@ def __init__(self,*, root):
         self._pq = [] # The heap queue
         self._entries = {} # {Block: Entry} mapping to look up if a block is already in the queue
         self._counter = itertools.count()
-        self._lock = threading.RLock()
-        self._update = threading.Condition()
+        self._clock = threading.Condition(threading.RLock())
```

Comment:
Maybe name this `self._cond`? It took me a sec the realize that `clock` is short for Condition Lock.

Reply:
Yeah I can rename. The main point is that you can use one object for all locking here. maybe condLock?